src/sandstorm/package.capnp

```diff
@@ -201,6 +205,163 @@ struct BridgeConfig {
   # "/" is a valid value, and will give clients access to all paths.
 }
 
+struct Metadata {
+  # Data which is not needed specifically to execute the app, but is useful for purposes like
+  # marketing and display.
+  #
+  # Technically, appMarketingVersion and appTitle belong in this category, but they were defined
+  # before MarketData became a thing.
+
+  icon :group {
```

Comment:
Well, originally I had `mainIcon`, `grainIcon`, `bannerIcon`. Then I defined a group basically for documentation purposes.

An `IconSet` type makes sense if there's some reason we'd expect it to be reused elsewhere or passed around independently of the surrounding Metadata object. I couldn't think of any reason to expect this, but arguably there's no good reason not to?